commands/generate.go

```diff
@@ -150,6 +151,9 @@ func runGenerate(cmd *cobra.Command, args []string) error {
 		if parsedServices != nil {
 			services = *parsedServices
 		}
+	} else {
+		fmt.Println("`stack.yml` file not found in the current directory.\nUse `--yaml` to pass a file or `--from-store` to generate using store image.")
+		os.Exit(0)
```

Comment:
In this instance can you exit 1? In the way, the program signals to its caller that there was an error.

Reply:
okay!